src/app/components/signup/signup.component.html

```diff
@@ -0,0 +1,69 @@
+<!-- registration.component.html -->
+
+<div class="container mt-4">
+    <div class="row justify-content-center">
+      <div class="col-md-6">
+        <div class="card">
+          <div class="card-header">
+            <h3 class="mb-0">Register</h3>
+          </div>
+          <div class="card-body">
+            <form [formGroup]="registrationForm" (ngSubmit)="onSubmit()">
+              <div class="form-group">
+                <label for="email">UserName<span class="text-danger">*</span></label>
+                <input type="email" formControlName="username" class="form-control">
```

Comment:
I think we can use type text for username instead of email

Reply:
yeah thanks forgot to change